packages/components/src/draggable/index.native.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { Platform } from 'react-native';
```

Comment:
We might consider using the `Platform` object from `@wordpress/element` package ([reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/element/src/platform.ios.js)).

Reply:
Thanks for the suggestion! Updated in https://github.com/WordPress/gutenberg/pull/39551/commits/9afd4b911dc46fa3bd0422c25cf2d5ac6d38dcfd